src/components/seeds/database/ImportAccessionsModal.tsx

```diff
@@ -0,0 +1,354 @@
+import { Link } from '@mui/material';
```

Comment:
yes if you want to build out a reusable component that would be cool, if you want to do that as a fast follow-up so we can start testing, that's fine too

Reply:
Yes, lets merge this first and I will create a component for the two cases in next pr